laravel/tests/Unit/EvaluationParserTest.php

```diff
@@ -221,12 +223,11 @@ public function testGetDate()
     // Test user find people request
     public function testUserFindPeople()
     {
-        $result = null;
         try {
             $result = EvaluationParser::findPeopleOSU("Michael", "Bragalone");
+            $this->assertNotNull($result);
         } catch (\Exception $e) {
-            echo "\nTESTING AdminAddUserTest: Exception caught for find People OSU request for Michael Bragalone";
+            $this->expectException(\Exception::class);
         }
```

Comment:
what if we split this into two tests, one for successful find person and one for failure? Something like the following might suffice.

Additionally, can we be more specific about the type of exception we expect than just the general `Exception` class?

```php
public function testUserFindPeopleFailure()
{
    $this->expectException(\Exception::class);
    $result = EvaluationParser::findPeopleOSU("NotA", "Person");
}

public function testUserFindPeopleSuccess()
{
    $result = EvaluationParser::findPeopleOSU("Bob", "Smith"); // change this name to someone who we KNOW exists
    $this->assertNotNull($result);
}
```

Reply:
I added the try catch because sometimes guzzle http has problems - these are not expected and unfortunately guzzle has problems with these connection failures that many people have tried to find a way around with not much luck - these vague try catches seemed to be the best bet people have got. I was going to put try catches around medhub calls as well since we have that connection error sometimes. I am not deliberately making a faulty request to catch an exception, I'm just trying to avoid failure for guzzle issues (not the actual request)

Reply:
In that case, I misunderstood what the try catch was for.

Error handling really belongs in application code, not tests. If anything goes wrong with the application during testing, the test status should reflect that.

Suppressing guzzle's failures with the try catch is a dangerous idea. OSU Find People could go down, but the test would keep working.

Reply:
How I had it originally 
```$result = NULL;
        try{
            $result = EvaluationParser::findPeopleOSU("Michael", "Bragalone");
        }catch (\Exception $e){
            echo "\nTESTING AdminAddUserTest: Exception caught for find People OSU request for Michael Bragalone";
        }
	$this->assertNotNull($result);
```

would not suppress the failure - the test would still fail. The code does have try catches. They are not within the method itself.

Reply:
Can we put a pin in this until a zoom meeting? I don't feel great about how we're handling this, but don't have any better ideas right now.

Reply:
Put some more thought into this. I'm okay with it if we include a `$this->fail('message');` in the catch branch.

Alternatively, we could use `$this->markTestSkipped('message.');`, but I think failure is more appropriate.